blockchain/processing.py

```diff
@@ -606,9 +626,85 @@ def _execute_phase_4(self, config, phase_3_info):
 
             print "phase 4 executed"
 
-    def _execute_phase_5(self, config, phase_5_info):
+    def _execute_phase_5(self, config, verification):
         """ public, Bitcoin bridge phase """
-        print "phase 5 executed"
+        phase = 5
+        verification_record = verification['record']
+
+        verification_info = verification['verification_info']
+        verification_record['verification_info'] = verification_info
+
+        if validate_verification_record(verification_record, verification_info):
+            timestamp_db.insert_verification(verification_record)
+            verification_record['block_id'] = None
```

Comment:
Yes - IIRC, also that we may be placing multiple blocks (with different block IDs) from multiple nodes into this phase 5 record.

Can you comment to that effect for each of these lines? TODO